src/main/telemetry/telemetry.c

```diff
@@ -137,6 +137,9 @@ void telemetryCheckState(void)
 #ifdef TELEMETRY_SRXL
     checkSrxlTelemetryState();
 #endif
+#ifdef TELEMETRY_IBUS
+    checkIbusTelemetryState();
```

Comment:
Because the calls here where missing, the iBus code was probably not linked into the targets previously, thus leading to 0 size. Now that the code gets used, it does use some space.